### DISCUSSION TITLE
Is it possible to use sandpack for components declared in the code in node_modles and not in npm?

### DISCUSSION THREAD

**Discussion by taheramr** [codesandbox/sandpack#922] [General]
My use case is that I want to have a preview of sandpack with a playground that I can write code and import components declared inside of my node_modules, I don't want any network requests occurring from sandpack to codesandbox, only the preview of react and code editor is fairly enough.

**Comment by danilowoz** [comment]
Here's a few guides to achieve that: 
- [Self-host bundler](https://sandpack.codesandbox.io/docs/guides/hosting-the-bundler)
- [Local dependencies](https://sandpack.codesandbox.io/docs/guides/local-dependencies)

Let me know if it works
